Readme.md

```diff
@@ -41,9 +41,11 @@ import FroalaEditorComponent from 'react-froala-wysiwyg';
 // Import a language file.
 // import 'froala-editor/js/languages/de.js';
 
-// Include font-awesome css if required.

```

Comment:
Why did you removed font awesome guide? you should add one extra line here so that font awesome will work and should rearrange steps like below

// Include font-awesome css if required.
// install using "npm install font-awesome --save"
// import 'font-awesome/css/font-awesome.css';
// import 'froala-editor/js/third_party/font_awesome.min.js';